treeherder/webapp/api/performance_data.py

```diff
@@ -199,12 +199,13 @@ def list(self, request, project):
 
         datums = PerformanceDatum.objects.filter(
             repository=repository).select_related(
-                'signature__signature_hash').order_by('push_timestamp')
```

Comment:
This is probably not necessary to keep as its most likely used in graphs in the UI.